.travis.yml

```diff
@@ -15,8 +15,11 @@ install:
   - sudo apt-get install rethinkdb
   - pip install -e .[test]
   - pip install codecov
+  - pip install flake8
 
-before_script: rethinkdb --daemon
+before_script:
+  - rethinkdb --daemon
+  - flake8 --max-line-length 119 bigchaindb/
```

Comment:
I wonder where is the optimal place to put the check. Could we put it in `before_install` before the rethinkdb installation commands? That would save time when the flake8 command errors out. Would it?